core/lib/file-storage.js

```diff
@@ -0,0 +1,201 @@
+// Taken from https://github.com/VeliovGroup/Meteor-Files/blob/master/docs/aws-s3-integration.md
+import { Meteor } from 'meteor/meteor';
+import { _ } from 'meteor/underscore';
+import { Random } from 'meteor/random';
+import { FilesCollection } from 'meteor/ostrio:files';
+import stream from 'stream';
+
+import S3 from 'aws-sdk/clients/s3'; /* http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html */
```

Comment:
Looks like you forgot to add aws-sdk to the deps @Flowtter :)

Reply:
Fixed :)